src/definition/messages/IMessageFollowContext.ts

```diff
@@ -0,0 +1,21 @@
+import { IMessage } from '.';
+import { IUser } from '../users';
+
+/**
+ * The context of execution for the following events:
+ * - IPostMessageFollowed
+ */
+export interface IMessageFollowContext {
+    /**
+     * The message that was followed or unfollowed
+     */
+    message: IMessage;
+    /**
+     * The user who follow the message
+     */
+    user: IUser;
+    /**
+     * If the message was followed or unfollowed
+     */
+    isUnfollow: boolean;
```

Comment:
I think it would be better to be in the positive instead of being a double negative.

Reply:
I agree. Even though it might seem kinda _weird_ because the name of the event is `IPostMessageFollowed`, I think the double negative will be more confusing